Userland/Libraries/LibWebView/AriaPropertiesStateModel.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2030, Jonah Shafran <jonahshafran@gmail.com>
```

Comment:
The copyright year should be the current year (though it would be interesting to imagine what Ladybird would look like in 2030). This sneaky typo is prevalent in a few other files I have seen thus far.